genie-web/src/main/java/com/netflix/genie/web/data/services/PersistenceService.java

```diff
@@ -107,12 +107,12 @@
      * @return The page of found applications
      */
     Page<Application> findApplications(
-        @Nullable String name,
-        @Nullable String user,
-        @Nullable Set<ApplicationStatus> statuses,
-        @Nullable Set<String> tags,
-        @Nullable String type,
-        Pageable page
+            @Nullable String name,
```

Comment:
Hmm.. all of these got 8 spaces of indentation. Is that right..?

Reply:
Good catch. Not sure why this changed all of a sudden but the IDE `continuation indent` setting was 8 spaces.

Fixed